src/Traits/LoadsFilesTrait.php

```diff
@@ -0,0 +1,57 @@
+<?php
+namespace Michaels\Manager\Traits;
+
+/**
+ * Allows a class to have file loading capabilities.
+ *
+ * @implements Michaels\Manager\Contracts\ManagesItemsInterface
+ * @package Michaels\Manager
+ */
+
+use Michaels\Manager\Contracts\DecoderInterface;
+use Michaels\Manager\FileLoader;
+
+trait LoadsFilesTrait
+{
```

Comment:
be sure to `use Michaels\Manager\Traits\DependsOnManagesItemsTrait` which only ensures that the Trait isn't being used without `ManagesItemsTrait`. See `ChainsNestedItemsTrait` for an example.